Provider Provisioner/src/handle-input-offer.ts

```diff
@@ -28,19 +35,49 @@ export async function handleInputOffer(params: HandleInputOfferParams): Promise<
   if (!orders.length) {
     return;
   }
+  await spctlService.initialize();
 
   const resourceJsonPath = 'resource.json';
   await fs.writeFile(resourceJsonPath, JSON.stringify(inputOffer.resourceFileContent), 'utf-8');
 
+  await Promise.allSettled(
```

Comment:
```suggestion
  await Promise.all(
```
I noticed that you’re already catching all errors in your promises. In such cases, the use of `.allSettled` might not be necessary.